modules/301-prometheus-metrics-adapter/docs/USAGE_RU.md

```diff
@@ -133,17 +141,18 @@ spec:
     object:
       # Объект, который обладает метриками в Prometheus.
       describedObject:
-        apiVersion: extensions/v1beta1
+        apiVersion: networking.k8s.io/v1
         kind: Ingress
         name: myingress
       metric:
         # Метрика, зарегистрированная с помощью CR IngressMetric или ClusterIngressMetric.
+        # Можно использовать rps_1m, rps_5m или rps_15m которые поставляются с модулем prometheus-metrics-adapter.
         name: mymetric
       target:
-        # Для метрик типа Object можно использовать только `type: Value`.
-        type: Value
-        # Масштабирование, если значение кастомной метрики больше 10.
-        value: 10
+        # Для метрик типа Object можно использовать `Value` или `AverageValue`.
+        type: AverageValue
```

Comment:
The option with values is incorrect for RPS.
Because rps is the load on all pods, then at the moment the border was reached, the scale went to the maximum number of pods.
Because the value of the metric does not depend on the number of pods.
With the new option, the metric value is divided by the number of pods, which stops the increase in the number of pods when the value becomes less than the trigger threshold.
Also, the number of RPS is easier to calculate per 1 pod during load testing.